doc/vector/riscv-crypto-vector-zvkn.adoc

```diff
@@ -11,7 +11,7 @@ This extension is shorthand for the following set of other extensions:
 
 
 | Zvkned  | <<Zvkned>>
-| Zvknhb  | <<Zvknh>>
+| Zvknhb  | <<zvknh,Zvknhb>>
```

Comment:
It looks like the anchor might even be `[[zvknh, zvknh[ab]]]` or is this pointung to a different part ?

Reply:
I've noticed that the Zvkn - NIST Algorithm Suite explicitly requires **Zvknhb**, which encompasses both SHA-256 and SHA-512. Would marking the pointer with **Zvknh[ab]** introduce ambiguity or confusion?

Reply:
The pointer is an address in the document, not a mandate (`Zvknh[ab]` are defined simultaneously in the same section). I was afraid that the pointer was not going to work if not written properly.

Reply:
I see, I was following this example in riscv-crypto-vector-element-groups.adoc:
https://github.com/riscv/riscv-crypto/blob/651644fedfc80732104f5b56841de9d8a2d921b7/doc/vector/riscv-crypto-vector-element-groups.adoc?plain=1#L29

Reply:
Ok, it looks like this link works (tested from the latest pdf) so maybe I missed something.